opm/parser/eclipse/EclipseState/Schedule/Action/PyAction.hpp

```diff
@@ -75,6 +76,7 @@ class PyAction {
     RunCount m_run_count;
     std::string input_code;
     void * m_storage = nullptr;
+    mutable bool m_active = true;
```

Comment:
i would imagine this needs serialization (and a changed value in test object)? and comparison operator also.

Reply:
Well - a good question. *Iff* we assume that serialization support is for the initial distribution of stuff, and we do not envision serializing state throughout the simulation - the answer is no - the default value `active = true` set in the declaration will do.

Of course if we want to serialize as we go - things are differently. I can add serialization.

Reply:
we certainly do not plan to serialize as we go. we broadcast initial state then it evolves independently on processes. so if this can never be anything but true before simulator starts, it's all good. just knee jerking. i do think comparison operator should be updated though.